pages/projects-list/index.tsx

```diff
@@ -0,0 +1,202 @@
+import LoadingSpinner from "@/components/emptyStates/LoadingSpinner";
+import NoneFound from "@/components/emptyStates/NoneFound";
+import Body from "@/components/layout/Body";
+import ProjectTable from "@/components/tables/ProjectTable";
+import LoadingWrapper from "@/components/wrappers/LoadingWrapper";
+import NoDataWrapper from "@/components/wrappers/NoDataWrapper";
+import useCohort from "@/hooks/useCohort";
+import { isFetching } from "@/hooks/useFetch";
+import useInfiniteFetch, {
+  createBottomOfPageRef,
+} from "@/hooks/useInfiniteFetch";
+import { Cohort } from "@/types/cohorts";
+import { LEVELS_OF_ACHIEVEMENT_WITH_ALL, Project } from "@/types/projects";
+import {
+  Box,
+  debounce,
+  MenuItem,
+  Stack,
+  Tab,
+  Tabs,
+  tabsClasses,
+  TextField,
+} from "@mui/material";
+import {
+  ChangeEvent,
+  SyntheticEvent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+
+const LIMIT = 20;
+
+const ProjectsList = () => {
+  const [selectedLevelOfAchievement, setSelectedLevelOfAchievement] =
+    useState<LEVELS_OF_ACHIEVEMENT_WITH_ALL>(
+      LEVELS_OF_ACHIEVEMENT_WITH_ALL.ALL
+    );
+  const [page, setPage] = useState<number>(0);
```

Comment:
```suggestion
  const [page, setPage] = useState(0);
```

The type is implicit